### PR TITLE
CFDE Portal Tutorial (new), General tutorials format editing (fixes), social media icons & keys syntax (feature)

### LINKED ISSUE #209: CFDE Portal Tutorial(s)
Tutorial Requirements:
Consists of researcher use case (or another use case that can be completed using the portal) and write a tutorial for it.
The site goes public 10/31 so it needs to be in the 10/15 release
At this exact moment, focus on the walkthrough with limited screenshots, as the interface is currently being changed. We'll drop in screenshots last.

### PR DESCRIPTION
**Pull request by s-canchi**
## PR Checklist

- [x] Release label added
- [x] PR category label added
- [x] PR mergeable
- [x] Clean build logs
- [x] Linked related issues
- [x] Colorblindness test https://www.toptal.com/designers/colorfilter/

## PR Description
This PR includes the CFDE Portal use cases tutorial in the CF Tutorials tab. There are also included fixes to pages syntax (arrange --> nav), typographical and format edits along with addition of keys syntax across all tutorials that referenced keyboard keys.
Preview link - https://cfde-training-and-engagement--235.com.readthedocs.build/en/235/?next=https%3A%2F%2Fcfde-training-and-engagement--235.com.readthedocs.build%2Fen%2F235%2F&ticket=ST-1601963438-fcIVOoVsM20U0dEtrMyBeaX9iuIeFhBY

## Review format
Leave suggestions in comments and I will edit them. 

## Timeline
Urgent. The approved PR has to be merged by Oct 8th.

## Reviewer Checklist

- [ ] spelling and grammar
- [ ] successful run of all installation and code chunks
- [ ] sufficient explanation and details for the tutorial content
- [ ] code syntax and naming convention
- [ ] adherence of tutorial format to style guide
- [ ] functional links (inter and intra)
- [ ] accessible hyperlink text

### CODE REVIEW

**Comment by abhijna** [comment]
Nice work figuring all of this out! Here are my comments on the Functional Genetic Variation in Blood Cancer page.

- Video: [first thoughts] it's nice that you have a video. I think it would be more helpful to have audio guidance here to tell the user what step you're at. Or you could break it into three vidlets for each of the selection panels and pin it to the relevant step.
[later thoughts] The video was not very useful to me the first time I watched it, but it made more sense after I followed the written tutorial. Maybe if you moved it to the end, it would be better...? maybe ask others how they feel. 

- [x] step 2: can you put a red circle and arrow on the submit tab on the top right of the image. my instinct was to click on "next".
 
- [x] step 3: is there a reason why there is a red circle around the whole left panel? Can you put arrows to highlight only the two new boxes that you checked? and then add text in the step that the two blood boxes should have been checked already. If you could also put a circle around the number `7693` to show users where to look for this number, that would be super helpful.

- [x] step 4: can you make it more obvious somehow that the two selections are in different columns. GTEX is in `ID namespace` and the other one is in the `name` column. Maybe just clarify in the text.

- [x] step 5: you have to tell me where to click to export the `csv`. right now it's unclear where this spreadsheet suddenly appeared from. I'm assuming you clicked on the top left "Export" button and then clicked on "search result csv" option under it.
Also probably need to add a sentence somewhere here (before the next step) that the cvs only contains metadata -- one needs to go to the CF program's portal to get metadata. Maybe you can include this in an admonition (see next point).

- [x] step 6: you need an admonition here that users must have access to data and all they must have all their account linking done. You can link to setting up KF accounts page for the gen3 or whatever. 

- [x] step 6: under the upload a file option - do people have to make a new csv with only the identifiers or can they upload the whole csv that they downloaded from the portal? if they have to make a new csv, it might be easier to only give them the copy and paste option.

- [x] I would also recommend breaking up the tutorial into two pages: the first page about downloading the csv. second page about getting data from the kf portal and GTEx portal using the csv

Will post more comments on the next page soon.

**Comment by abhijna** [comment]
A lot of my comments on the first tutorial apply to this one as well. I see that you've started making changes already, so if you've addressed these comments, please ignore them here. 

- [x] like for the other page, it's probably worth highlighting where the user can find the number of files listed. 

- [x] Step 2: I think it's ok to call it a "search box". or put the "Search all columns" in quotes.

- [x] Step 3: make sure you indicate where the Export option is and that this is a two-step process. They are saving the csv to their local machine, yes? If so, remind them to take note of where they are storing the file on their local machines. Same for step 4. 

- [x] Step 5: I need a bit more context on what's going on here. Why am I merging cohorts? Where is the merging happening? Where did this spreadsheet come from? I wasn't asked to push any button. Unless I totally missed something. 

- [x] Steps 6 and 7: like with the other tutorial, there needs to be more information about what's happening here, why I'm doing this, etc. Consider moving to a different page or making it a different section on the same page instead of steps in the overall tutorial. Also, tell them they probably need protected data access to work with human biomedical data.

**Comment by s-canchi** [comment]
Thanks @abhijna for all the comments and suggestions! I have made all the necessary changes and updated fixes. You can check the latest version and approve if it looks good.

**Comment by marisalim** [comment]
@s-canchi this looks great! 2 main suggestions:

- [x] Let's say up-front either on the tutorial landing page or at the top of the data access pages that this tutorial isn't about getting the actual data files since those require applying for access, instead of putting that info at the end of the pages.
- [x] It's not clear why for the blood cancer section we are choosing data from KF and GTEx, and for the movement disorders, data from LINCS and Metabolomics. I assume for the KF and GTEx group, it would be suggesting a hypothetical study comparing cancerous (KF) and healthy (GTEx) tissue data? Can you add a sentence or two to the intros for these sections to explain? 

Minor edit
- [x] On [this](https://cfde-training-and-engagement--235.com.readthedocs.build/en/235/CF-Tutorials/CFDE-Portal/blood-cancer-data-access/) page, the link in the first sentence goes back to the previous page. is that right? it seems like it should open the manifest file, rather than the tutorial page, as written. Same for the movement disorders data access page. 
> Using the details in the CFDE portal exported csv manifest, 

I would rephrase to "Using the csv manifest details we just exported from the CFDE portal, "

**Comment by abhijna** [comment]
I added the help email to the footer. It's a bit small. I will look into making the icons bigger
(didn't want to open a new PR)

**Comment by s-canchi** [comment]
@marisalim thank you for the suggestions! I have fixed all of them in the latest version. With regards to the link in the data access page it is correct that it takes them to the previous page because it details the steps to get the csv file. Since the tutorial is about using the portal, I intentionally did not include the exported csv files to the users.

**Comment by abhijna** [comment]
@s-canchi I think the new pages that you made based on my suggestions need to be nested pages. So the two tutorial that you split should be in their own folders (if that makes sense). that might clear marisa's confusion.

**Comment by marisalim** [comment]
@s-canchi how about this for the data access note?

> After generating the metadata manifest file for our cohort, we can search for these particular datasets on their respective Common Fund data portals. While some datasets may be public-access, many are likely to be controlled-access. Our tutorial cohort includes controlled-access data. Below, we show how a researcher would 1) use the metadata manifest information on CF portals and 2) apply for access to protected data.

**Comment by s-canchi** [comment]
Both suggestions are great! I will reorganize a bit more and hopefully arrive at the final version :)

**Comment by ACharbonneau** [comment]
for the data access note, you might want to consider keeping the general message, but making it about the *user* rather than about *us*. Those beginning of tutorial blurbs are supposed to get the user interested in participating, so they should be from their perspective. There's also a *lot* of words in here that are overly big, and that your users aren't going to know. So something more like:

> By following this tutorial researchers and clinicians will learn to navigate within the Common Fund Data Ecosystem Data Portal. Users will learn how to use this portal to find datasets from many different Common Fund Programs in a single search to build a cohort, and to save that cohort as a single manifest file. While some discoverable datasets may be public-access, many are likely to be controlled-access. Researchers will learn how to apply for protected data access for files included in their manifest.

For the paragraph above that, same problem: assuming too much background knowledge, and you've linked to the technical docs, which are for *model developers* instead of the user docs. Try something like:

> The NIH Common Fund has funded a wide variety of data types and studies that are of interest to clinical and biomedical researchers, however those datasets are hosted on an equally large number of websites, with varying query systems. The Common Fund Data Ecosystem (CFDE) Data Portal is a unified system for searching across the entire Common Fund portfolio in a single search, and is the first step in addressing the goal of making Common Fund data more [Find-able, Accessible, Interoperable and Reusable (FAIR)](https://www.nih-cfde.org/product/fair-cookbook/). The wide range of data types, models and formats used by Common Fund Programs are being harmonized using well-defined metadata, common controlled vocabularies using the [Crosscut Metadata Model](https://www.nih-cfde.org/product/cfde-c2m2/).

**Review by marisalim — APPROVED** [review]
lgtm

**Review by abhijna — APPROVED** [review]
